### DISCUSSION TITLE
will collaborators contributions in a project be counted after getting merged

### DISCUSSION THREAD

**Discussion by sahildkun** [Spectrum-CETB/Spectober_Fest#62] [Q&A] [answered]
like in the netflix ui clone if me and other friends collaborate and if it gets merged then will the hacktober merge requests update like form 0/4 to 1/4 of each collaborator even though the pr is filled by a single person but has collaborators

**Comment by Aditya-150** [comment, marked as answer]
No! The pr actually will be counted for the one who made it. Not for all other contributors.
I hope your doubt is cleared. If so then pls give 👍 to the comment.

**Reply by Aditya-150** [reply]
Please `mark as answer` if it was helpful🙂

**Reply by sahildkun** [reply]
thanks

**Reply by Aditya-150** [reply]
> thanks

Happy🙂 to know that you are getting your doubts cleared using discussions.
Pls tell your friends to ask their doubts here.

We will appreciate it if do so.❤️
